.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,109 @@
+name: Bug Report
+description: File a bug report
```

Comment:
```suggestion
description: Open a bug report
```

Reply:
For some non-English speakers opening an issue might be easier to understand than filing it, but that's just minor suggestion.

Reply:
Yes good catch updated.